packages/experimental/Icon/src/Icon.tsx

```diff
@@ -56,16 +56,20 @@ function renderSvg(iconProps: IconProps) {
   const style = mergeStyles(iconProps.style, rasterImageStyleCache({ width: width, height: height }, [width, height])[0]);
 
   // react-native-svg is still on 0.61, and their color prop doesn't handle ColorValue
+  // If a color for the icon is not supplied, fall back to white or black depending on appearance
+  const theme = useTheme();
+  const iconColor = svgIconProps.color ? svgIconProps.color : theme.host.appearance === 'dark' ? '#FFFFFF' : '#000000';
```

Comment:
If the theme doesn't set it's own appearance or host, do we need to default to use the react-native appearance API?

Reply:
Thanks for the suggestion, I added that functionality. There's only so much we can do to try and support a fallback - worst case you'll get black, which is what you'd get if you didn't supply a color anyway.

Reply:
That's fair. Going through one hoop of checking the theme's appearance is probably enough. Thanks for looking into that!